src/llfs/ioring_impl.cpp

```diff
@@ -49,12 +49,14 @@ StatusOr<i32> status_or_i32_from_uring_retval(int retval)
 /*static*/ auto IoRingImpl::make_new(MaxQueueDepth entries) noexcept
     -> StatusOr<std::unique_ptr<IoRingImpl>>
 {
+  LLFS_VLOG(1) << "Creating new IoRingImpl";
```

Comment:
Is LLFS_VLOG better to use in LLFS than VLOG?

Reply:
Yes; in src/llfs/logging.hpp, you can see there are some alternative implementations for logging (including no logging at all).  Using the `LLFS_*` macros  lets you swap out a different logging layer.